package.json

```diff
@@ -66,5 +66,24 @@
   },
   "cypress-cucumber-preprocessor": {
     "nonGlobalStepDefinitions": true
+  },
+  "description": "LinkFree Connect audiences to all of your content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript - It was initially created on a YouTube [live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).",
+  "main": "generate.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/EddieHubCommunity/LinkFree.git"
+  },
+  "keywords": [
+    "javascript",
```

Comment:
Umm, not sure if this needs to be added especially when this file is a package.json

```suggestion
```